src/test/java/uk/gov/companieshouse/api/accounts/transformer/CurrentPeriodTransformerTest.java

```diff
@@ -39,6 +40,12 @@ public void testRestToEntityTransformerWithPopulatedObject() {
         BalanceSheet balanceSheet = new BalanceSheet();
         balanceSheet.setCalledUpShareCapitalNotPaid(5);
 
+        FixedAssets fixedAssets = new FixedAssets();
+        fixedAssets.setTangible(10);
```

Comment:
You could have `private static final Integer`'s at the class - one for each field - which are all different, which may improve readability. More a suggestion than a request though.